### ISSUE TITLE
Allow specific keywords to be blacklisted from alignment

### ISSUE THREAD

**Issue by jfelchner** [bbatsov/rubocop#1031]
As I mentioned in my previous issue regarding documentation, I prefer to namespace my classes like this:

```ruby
module Foo
class  Bar
  # Codez
end
end
```

There are specific advantages to doing it this way.  Because I always do this though, if I indented every level (sometimes it ends up being 3 or 4 levels) and I wrap my code (preferably) at 80 columns, the indentions cause the "usable space" for my code to be much smaller. So instead, I do not indent `module` or `class` lines but only start indenting in the body of the class.

I'd like an alignment option so that I can say "ignore lines that start with `module` or `class` (or any other arbitrary keyword like `describe`)".

**Comment by alexdowad** [comment]
We all have our own coding style preferences. I don't think that RC can (or should) cater to every possible style that anyone might want to use, though. That is why RC allows you to add your own custom cops.

Hopefully before long, the API for extending RC will be documented. You can even package up your custom extensions as a gem, and require it in all your projects.

I think that this issue should be closed.

**Comment by bbatsov** [comment]
Agreed.

**Comment by jfelchner** [comment]
@alexdowad so you're saying I can externally add this functionality to this cop myself?

**Comment by alexdowad** [comment]
@jfelchner, You can copy the source of the cop, modify it as desired to make a custom cop, load your custom cop into RC, then disable the original and enable the custom one.

**Comment by alexdowad** [comment]
(Of course, change the name as well.)
